app/labor/assign_tag_moderator.rb

```diff
@@ -1,63 +1,73 @@
 module AssignTagModerator
-  def self.add_trusted_role(user)
-    return if user.has_role?(:trusted)
-    return if user.has_role?(:banned)
-
-    user.add_role :trusted
-    user.update(email_community_mod_newsletter: true)
-    MailchimpBot.new(user).manage_community_moderator_list
-    Rails.cache.delete("user-#{user.id}/has_trusted_role")
-    NotifyMailer.with(user: user).trusted_role_email.deliver_now
-  end
+  class << self
+    def add_trusted_role(user)
+      return if user.has_role?(:trusted)
+      return if user.has_role?(:banned)
+
+      user.add_role :trusted
```

Comment:
**nitpick (non-blocking):** Can we add the parenthesis here? All the surrounding code has them.